misc/python/materialize/mzcompose/__init__.py

```diff
@@ -93,6 +93,7 @@ def get_default_system_parameters(
         "enable_continual_task_create": "true",
```

Comment:
Yes, this is the old name apparently. We should still keep it for when we run upgrades.

Reply:
Oh ooof.  Yeah, I renamed it for consistency with the other (new) continual task configs and I should have thought of this. Sorry!

Reply:
Not a big deal, so no worries.